packages/run-protocol/src/econ-behaviors.js

```diff
@@ -233,6 +250,7 @@ export const startVaultFactory = async (
 
   const vaultFactoryTerms = makeGovernedTerms(
     priceAuthority,
+    oraclePriceRatio,
```

Comment:
Making `oraclePriceRatio` governed seems out of scope here. That's a much bigger policy decision.

Please let's just make it an ordinary immutable contract term.

Otherwise, the stakeholders that need to approve this are more than just @turadg and I, I think.

Reply:
I don't see how to transmit it to the vaultFactory without it being in vaultFactoryTerms.  I'm hoping somebody else can advise on how to refactor this if necessary.

Reply:
sorry; my concern here is misplaced. `makeGovernedTerms` doesn't mean "make a collection of governed terms". It means "make the terms for this contract, including some that are governed." So it doesn't turn oraclePriceRatio into something that the electorate can change.